### PR TITLE
Allow sctp protocol with LatencySensitive FG

### PR DESCRIPTION
**Pull request by karmab**

### CODE REVIEW

File: config/v1/types_feature.go [hunk 1]
```diff
@@ -131,6 +131,7 @@ var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
 			"TopologyManager",                // sig-pod, sjenning
 			"NodeDisruptionExclusion",        // sig-scheduling, ccoleman
 			"ServiceNodeExclusion",           // sig-scheduling, ccoleman
+			"SCTPSupport",
```
Comment (deads2k) [inline config/v1/types_feature.go]: you need to provide a sig (or openshift team) and the name of the maintainer we will nag if it breaks or if we decide to remove it.

Comment (fedepaol) [reply config/v1/types_feature.go]: @karmab I think we can take inspiration from what was done with the default policy

**Comment by knobunc** [comment]
@deads2k @sttts @mfojtik -- What do we need to do to push this forward.  This is needed for important use-cases (see associated bug for details).  Thanks

**Comment by deads2k** [comment]
> @deads2k @sttts @mfojtik -- What do we need to do to push this forward. This is needed for important use-cases (see associated bug for details). Thanks

it'll need to be vendored where it's used now.

**Comment by fedepaol** [comment]
> > @deads2k @sttts @mfojtik -- What do we need to do to push this forward. This is needed for important use-cases (see associated bug for details). Thanks
> 
> it'll need to be vendored where it's used now.

Will do it. It was done before for the same reason here https://github.com/openshift/cluster-kube-apiserver-operator/pull/658

**Pull request merged**